webapp/locales/en.json

```diff
@@ -107,7 +107,7 @@
     "follow": "Follow",
     "followers": "Followers",
     "following": "Following",
-    "shouted": "Recommended",
```

Comment:
Because on the post it is named shout and I prefer to have it all the same. @roschaefer 
Recommend on posts is okay with me, but everywhere the same please.

Reply:
We have had this discussion before @roschaefer on Discord, and I certainly understand some would see it as bad naming, but I don't agree. 
There are many expressions in English that make reference to the meaning of `shout` as in to promote. 
See https://idioms.thefreedictionary.com/shout

Adding translations involves knowing the languages well enough to know popular expressions and their translations in the target language, so I don't see it as an issue that it would be slightly different from one language to the next.

Of course, people can disagree and there might be something to be said about the use of this word if a large quantity of the user base does not know English well enough to immediately understand what is meant to be communicated. :woman_shrugging: